immut/list/list.mbt

```diff
@@ -266,7 +266,6 @@ pub fn[A] unsafe_head(self : T[A]) -> A {
 }
 
 ///|
-#deprecated("Use `unsafe_head` instead")
 #coverage.skip
```

Comment:
shall we detect duplicated deprecations? @Yoorkin

Reply:
I think yes. The overlapping warnings will be shadowed, meaning some deprecation messages will not be displayed. This behavior might confuse users.